app/logic/volunteers.py

```diff
@@ -92,3 +98,26 @@ def setUserBackgroundCheck(user,bgType, checkPassed, datePassed):
     update = BackgroundCheck.create(user=user, type=bgType, passBackgroundCheck=checkPassed, datePassed=datePassed)
     update.save()
     createLog(f"Updated {user.firstName} {user.lastName}'s background check for {bgType} to {bool(checkPassed)}.")
+
+def isProgramManagerForEvent(user, event= None):
```

Comment:
Since you are not using program id here and event is required, you can remove the default value.